common/algorithm-terms.html

```diff
@@ -35,15 +35,15 @@
     but present in the <a>input frame</a>,
     should be omitted from the output.</dd>
   <dt class="changed"><dfn>omit graph flag</dfn></dt><dd class="changed">
-    A flag that determines if framing output is always contained within a <code>@graph</code> member,
+    A flag that determines if framing output is always contained within a <code>@graph</code> <a>entry</a>,
     or only if required to represent multiple <a>node objects</a>.</dd>
   <dt><dfn>processor state</dfn></dt><dd>
     The <a>processor state</a>,
     which includes the <a>active context</a>, <a>active subject</a>, and <a>active property</a>.
     The <a>processor state</a> is managed as a stack with elements from the previous <a>processor state</a>
     copied into a new <a>processor state</a> when entering a new <a>JSON object</a>.</dd>
-  <dt><dfn>promise</dfn></dt><dd>
-    A <a data-cite="ECMASCRIPT#sec-promise-objects" class="externalDFN">promise</a> is an object that represents the eventual result of a single asynchronous operation.
+  <dt><dfn data-cite="ECMASCRIPT#sec-promise-objects">promise</dfn></dt><dd>
+    A <a>promise</a> is an object that represents the eventual result of a single asynchronous operation.
```

Comment:
Note, this definition is not quite correct. This is just one property of a promise, but it's somewhat inaccurate (a promise can be immediately resolved/rejected, for instance)... if you can drop this redefinition entirely, it might be better (people can get the formal  definition  by visiting the WebIDL spec, which will lead them  to the ECMAScript spec).   

```suggestion
    A <a>promise</a> is an object that represents the eventual result of a single asynchronous operation.
```